recipes/jalview/meta.yaml

```diff
@@ -0,0 +1,32 @@
+
+about:
+    home: http://www.jalview.org/
+    license: GNU GENERAL PUBLIC LICENSE, Version 3.0
+    summary: |
+        Jalview is a free program for multiple sequence alignment editing, visualisation and analysis. 
+        Use it to view and edit sequence alignments, analyse them with phylogenetic trees and principal 
+        components analysis (PCA) plots and explore molecular structures and annotation.
+package:
+    name: jalview
+    version: 2.10.2b2
+
+build:
+  number: 0
+
+source:
+    fn: jalview_2_10_2b2.tar.gz
+    url: http://www.jalview.org/source/jalview_2_10_2b2.tar.gz
+    sha256: 0012b2e14274f3aeb17d224e947f6342336049357a909c900226fe77fe4b33c0
+
+requirements:
+  build:
+    - openjdk >=7
+    - ant
+    - xorg-libxtst
+  run:
+    - openjdk >=7
+    - xorg-libxtst
+
+test:
+    commands:
+      - jalview -h
```

Comment:
Do you need to run the tests also headless?

Reply:
? Not sure i get what you mean?!

Reply:
But i changed the call, seems the argument was wrong, lets see...